### PR TITLE
Normalize to controlled terms for `dct_format_s` and `gbl_resourceType_sm`

### PR DESCRIPTION
**Pull request by ghukill**
### Purpose and background context
Two fields written to the outputted MITAardvark records, `dct_format_s` and `gbl_resourceType_sm`, did not have their values controlled to suggested values from the Aardvark schema.  This was revealed when attempting to map facet filters from the geo TIMDEX UI that rely on these fields.

While updates are still required in Transmogrifier and the TIMDEX data model for where these values end up, normalizing them to controlled terms will benefit the quality of the data for facet aggregations.

How this addresses that need:
* sets of controlled terms have been added to records.controlled_terms
* method `SourceRecord.get_controlled_dct_format_s_term()` created to normalize values from source metadata
* method `SourceRecord.get_controlled_gbl_resourceType_sm_terms()` created to normalize values from source metadata
* these two new methods applied to `FGDC`, `ISO19139`, `GBL1`, and `Aardvark` source classes

### How can a reviewer manually see the effects of these changes?

The newly [added tests here](https://github.com/MITLibraries/geo-harvester/pull/171/commits/3a7dded89c34b17c82d41af6f52f8ca5a97d2a5a#diff-954aa20ba8b90565619fd2376342ad57a7c204129fa40ed9f457703ed5e241f6R327-R400) demonstrate how values coming from the field methods `_dct_format_s()` and `_gbl_resourceType_sm()` on _all_ `SourceRecord` classes are run through these cleaning and mapping methods.

### Includes new or updated dependencies?
NO

### Changes expectations for external applications?
NO

### What are the relevant tickets?
- https://mitlibraries.atlassian.net/browse/GDT-195

### Developer
- [X] All new ENV is documented in README
- [X] All new ENV has been added to staging and production environments
- [X] All related Jira tickets are linked in commit message(s)
- [X] Stakeholder approval has been confirmed (or is not needed)

### Code Reviewer(s)
- [x] The commit message is clear and follows our guidelines (not just this PR message)
- [x] There are appropriate tests covering any new functionality
- [x] The provided documentation is sufficient for understanding any new functionality introduced
- [x] Any manual tests have been performed and verified
- [x] New dependencies are appropriate or there were no changes

### CODE REVIEW

File: harvester/records/record.py [hunk 1]
```diff
@@ -228,6 +232,78 @@ def is_deleted(self) -> bool:
             return True
         return False
 
+    def get_controlled_dct_format_s_term(self, value: str | None) -> str | None:
```
Comment (ehanson8) [inline harvester/records/record.py:235]: I'm assuming you've done some analysis of the original vs the controlled term, not looking for hard numbers but what was your general sense of the accuracy of these crosswalks?

File: tests/test_records/test_record.py [hunk 2]
```diff
@@ -322,3 +322,79 @@ def test_record_decode_doubled_encoded_source_json_data_success():
...
+
+def test_controlled_format_value_direct_match(generic_source_record):
+    assert generic_source_record.get_controlled_dct_format_s_term("GeoTIFF") == "GeoTIFF"
+
+
+def test_controlled_format_value_miss(generic_source_record):
+    assert generic_source_record.get_controlled_dct_format_s_term("watermleon") is None
+
+
+@pytest.mark.parametrize(
```
Comment (ehanson8) [inline tests/test_records/test_record.py:335]: Nice approach, haven't used this before!

**Review by ehanson8 — APPROVED** [review]
Nice set of tests and the code looks good! A question but approved

File: harvester/records/record.py (hunk 1 above)
Comment (ghukill) [reply harvester/records/record.py:235]: Good question.  It's accurate, particularly for MIT data.  The field this pulls from in FGDC (ISO is not a problem) returns maybe 10-20 unique values, but this mapping here captures the various forms and maps to controlled terms for us.

I should have kept the examples/analysis, but I think they're gone to the wind, but the mapping basically covers the values seen for our data sans a couple of really strange little stragglers.

**Comment by ghukill** [comment]
As mentioned in slack, I've pushed a new commit that adds some additional logic to produce a `dct_format_s` value: https://github.com/MITLibraries/geo-harvester/pull/171/commits/59b0b14ec3e97cd9c8ba8dbb27a120d213ead5f6.  

@ehanson8, I've re-requested a review, but please feel free to only review this commit.

**Review by ehanson8 — APPROVED** [review]

**Review by jonavellecuerdo — APPROVED** [review]

**Pull request merged**